dev/tools/lib/globals.dart

```diff
@@ -33,3 +37,52 @@ class ConductorException implements Exception {
   @override
   String toString() => 'Exception: $message';
 }
+
+Directory _flutterRoot;
+Directory get localFlutterRoot {
```

Comment:
This was part of the `Repository` class before, but it was lifted out to a global method so that it could be overridden in unit tests.